### PR TITLE
Hyper-kinetics now have splash damage

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Hyper-kinetic accelerators now have splash damage, which is 33% of its normal damage and will not hit the firer, but only fire as fast as a normal kinetic accelerator.
tweak: Super-kinetic accelerators fire like a tenth of a second faster.
bugfix: Hyper-kinetic shots that directly strike gibtonite will no longer cause it to explode.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/projectile/special.dm [hunk 1]
```diff
@@ -160,23 +154,37 @@
...
 
-/obj/effect/kinetic_blast/New()
-	spawn(4)
-		qdel(src)
+/obj/item/projectile/kinetic/hyper/proc/aoe_blast(atom/target)
+	var/turf/target_turf = get_turf(target)
+	if(!target_turf)
+		target_turf = get_turf(src)
+	PoolOrNew(/obj/effect/overlay/temp/explosion, target_turf)
+	for(var/turf/T in range(1, target_turf))
```
Comment (RemieRichards) [inline code/modules/projectiles/projectile/special.dm]: istypeleless+```RANGE_TURFS()```

**Comment by KorPhaeron** [comment]
@RemieRichards 

please put your avatar back change scares me

**Comment by RemieRichards** [comment]
@KorPhaeron it scares you here? you should see it on the forums >:)

**Comment by KorPhaeron** [comment]
hypers need some sort of downside or tradeoff instead of super range, extra damage, aoe damage

**Comment by ChangelingRain** [comment]
What would you suggest? I'd probably say 'slower fire time' because that seems fair, you get more damage and fancy AOE but can't fire as often.

**Comment by KorPhaeron** [comment]
I forgot they fire twice as often because by thr time you have hypers you have mod kits

**Comment by ChangelingRain** [comment]
Hypers will no longer fire faster because kor wants me to balance meme and this is what we get

**Comment by Kromgar** [comment]
TY for fixing the gibtonite that shit is the worst dieing in full drake armor because you hit a gibtonite was shit. Losing limbs and shit  from an explosion shouldnt be possible in drake armor

**Comment by KorPhaeron** [comment]
too small of a downside to buff the best mining item even further sorry

**Pull request closed without merging**